lib/browser/api/net.ts

```diff
@@ -409,6 +410,10 @@ export class ClientRequest extends Writable implements Electron.ClientRequest {
       return ret;
     };
     this._urlLoaderOptions.referrer = this._urlLoaderOptions.extraHeaders.referer || '';
+    this._urlLoaderOptions.origin = this._urlLoaderOptions.origin || this._urlLoaderOptions.extraHeaders.origin || '';
```

Comment:
Headers are case insensitive when read by the networking stack but this assumes that `origin` is lowercase here.  To be accurate this should somehow pull `origin` in a case insensitive way right?

Reply:
The header name are stored in lowercase internally. This is another issue we may fix in the future.

```
  setHeader (name: string, value: string) {
    ...
    const key = name.toLowerCase();
    this._urlLoaderOptions.extraHeaders[key] = value;
  }
```

Reply:
I just go ahead and fix this issue too.